packages/ui-components/src/components/UICreateSelect/UICreateSelect.tsx

```diff
@@ -164,8 +156,8 @@ export const UICreateSelect: FC<UICreateSelectProps> = (props: UICreateSelectPro
             allowCreateWhileLoading={true}
             formatOptionLabel={(option) => option.label}
             placeholder={props.placeholder}
-            onChange={props.handleOnChange}
-            onCreateOption={props.handleCreate}
+            onChange={props.onChange}
+            onCreateOption={props.onCreateOption}
             options={props.options}
             value={props.value}
             noOptionsMessage={props.noOptionsMessage}
```

Comment:
if we extend from `props` of `react-select`. I think it is easier to spread props to avoid manual assignment:
```
ref={props.creatableRef}
            {...props}
            className="ui-create-select"
            classNamePrefix="ui-create-select"
            openMenuOnFocus={false}
            createOptionPosition="first"
            allowCreateWhileLoading={true}
            formatOptionLabel={(option) => option.label}
            formatCreateLabel={formatCreateLabel}
            components={{ ClearIndicator, DropdownIndicator, LoadingIndicator, Option }}
            unstyled={true}
```